modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/emodelrelease/v0.1.1.json

```diff
@@ -7,9 +7,7 @@
   ],
```

Comment:
I suggest to bump the version of this schema and perform the necessary changes their.